Assets/Scripts/Game.cs

```diff
@@ -4,4 +4,15 @@
 
 public class Game : MonoBehaviour
 {
+    public GameObject CardPrefab;
```

Comment:
imo `[SerializeField]` is better approach

Reply:
Will check